Fundamentals/Backoffice/Login/index-v9.md

```diff
@@ -0,0 +1,100 @@
+---
+meta.Title: "Configure and customize the Login screen"
+meta.Description: "In this article you can learn the various ways of customizing the Umbraco backoffice login screen and form."
+versionFrom: 9.0.0
+---
+
+# Login screen
+
+To access the backoffice, you will need to login. You can do this by adding `/umbraco` at the end of your website URL, e.g. http://mywebsite.com/umbraco.
+
+You will be presented with a login form similar to this:
+
+![Login screen](images/backoffice-login.png "The login screen has a greeting, username/password field and optionally a 'Forgotten password' link.")
+
+The **login** screen contains a **Greeting**, **Email**, **Password** field and optionally a **Forgotten password** link
+
+Below, you will find instructions on how to customise the login screen.
+
+## Greeting
+
+The login screen features a greeting, which you can personalize by changing the language file of your choice. For example for en-US you would add the following keys to: `~/umbraco/config/lang/en_us.xml`
+
+```xml
+<area alias="login">
+    <key alias="greeting0">Happy super Sunday</key>
+    <key alias="greeting1">Happy manic Monday </key>
+    <key alias="greeting2">Happy tubular Tuesday</key>
+    <key alias="greeting3">Happy wonderful Wednesday</key>
+    <key alias="greeting4">Happy thunderous Thursday</key>
+    <key alias="greeting5">Happy funky Friday</key>
+    <key alias="greeting6">Happy Caturday</key>
+</area>
+```
+
+You can customize other text in the login screen as well, grab the default values from `~/Umbraco/Config/Lang/en.xml` and copy the keys you want to translate into your `~/Umbraco/Config/Lang/MYLANGUAGE.xml` file.
+
+## Password reset
+
+The **Forgotten password?** link allows your backoffice users to reset their password. To use this feature, you will need to add the following key to the `"security"` section in the `appsettings.json` file:
+
+```json
+"Security": { 
+        "AllowPasswordReset": true
+      },
+```
```

Comment:
I think it makes sense if we for each of these configs, show all the groups/sections as well.

So, it would be like this instead:

```
"Umbraco": {
    "CMS": {
      "Security": { 
        "AllowPasswordReset": true
      }
   }
}

It shows all sections, and then the key with the correct value.
What do you think of doing it like that instead?
```